pczt/src/lib.rs

```diff
@@ -1 +1,135 @@
+//! The Partially Created Zcash Transaction (PCZT) format.
+//!
+//! General flow for creating a shielded transaction:
+//! - Create "unsigned transaction"
+//!   - In practice means deciding on the global parts of the transaction
+//! - Collect each output
+//!   - Proofs can be created at this time
+//! - Decide on an anchor
+//!   - All spends should use the same anchor for indistinguishability
+//!   - In a future transaction version, all spends will be required to do so
+//! - Collect each spend
+//!   - Proofs can and should be created at this time
+//! - Create proofs for each spend and output
+//!   - Data necessary for proofs can be stripped out of the format
```

Comment:
Are these actually separate steps?

Reply:
I've removed this content; it was only informational and not kept up-to-date.